lambda/smsUtil.js

```diff
@@ -0,0 +1,36 @@
+const AWS = require('aws-sdk');
+const { gcnSend } = require('./gcNotifyUtils');
+const { sendResponse } = require('./responseUtil');
+const { logger } = require('./logger');
+
+
+exports.sendSMSMessage = async function (payload, cancellationLink){
+    try {
+       let gcnSendObj = {
```

Comment:
`const` not `let` since you are not modifying it later.

Reply:
fixed in commit [0f1d4ee](https://github.com/bcgov/parks-reso-api/pull/330/commits/0f1d4eebe2a175a7de826b38d3a73f2b71648b2b)